aws.yaml

```diff
@@ -8,4 +8,4 @@ role: arn:aws:iam::123456789000:role/role-name
 aws_id: AKIAHI38FAKE1IWUQEEN
 aws_key: PA3XsxZ8d8cPQLmnZzFAKEdzC6ND2a8vhbyXU/Dw
 aws_token: FakeYXdzELv//////////wEldj3948yOJRO84jgpoip239232hEOHhfkjhefkwue97jorhfiuh+XjFC9Je/YG7JCqKjrspab2lB+7/Fb1NJFjgwur47Dbhs/L7nh+/VGnwLoAo8CIqoPBLRmXItaoiuuofZnr+ktihZk1Yi55sYZ12hfRMPVbDmhf9Ke683+e9bJirhUEghw9424JOhgwrgqq99MvzCEFe4eXPOSgAcQcD2xqnnKO738tjhoh23HFqjflhefibWegfqefgqUF12hvgfwegqf
-aws_id: AKIAHI38FAKE1IWUQEEXW
+aws_id: AKIAHI38FAKE1IWUQEEXP
```

Comment:
🛑 **Gitleaks** has detected a secret with rule-id `aws-access-token` in commit 6ecc76070473411ec6ed22805fb53b8a6e1705b8.
If this secret is a _true_ positive, please rotate the secret ASAP.

If this secret is a _false_ positive, you can add the fingerprint below to your `.gitleaksignore` file and commit the change to this branch.

```
echo 6ecc76070473411ec6ed22805fb53b8a6e1705b8:aws.yaml:aws-access-token:11 >> .gitleaksignore
```


cc @smandal-tharkka